__tests__/Snapshottest/UIOutputComparisonTests.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as LibraryEntryPoint from '../../src/entry-point';
+import { LibraryContainer } from '../../src/components/LibraryContainer';
+import { LibraryItem } from '../../src/components/LibraryItem';
+import { ItemData } from "../../src/LibraryUtilities";
+import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
```

Comment:
@fanwgwg  Its already there in the above commit - https://github.com/DynamoDS/librarie.js/pull/89/commits/26a1bb5f8c8354ebc0532fae073fd2be115d4541

![image](https://user-images.githubusercontent.com/5218721/27164565-ba47761e-51c0-11e7-861d-e9cabca136bd.png)